### PR TITLE
Download button

### PR DESCRIPTION
**Pull request by sciencefixion**
### New Feature Submissions:
* [X] Have you added an explanation of what your changes do?
* [X] Have you linted your code locally prior to submission?

### Testing:
* [X] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed?

**Test coverage** = 100%
--------------------------------------------------------------------------------
### Change Log:

* Added a redirect to the user dashboard if a user is already logged in
* Added a download button to the search show page
* Styled developers index page
* Moved Glitchable to modules folder
* Added the SecureRandom require statement to ImagesController for future use with naming files

### CODE REVIEW

File: app/controllers/images_controller.rb [hunk 1]
```diff
@@ -13,6 +15,13 @@ def create
     redirect_to search_show_path(api_image)
   end
 
+  def download
+    # uuid = SecureRandom.uuid
+    # image_file = "lib/assets/new_image.png"
+    # File.rename(image_file, "lib/assets/glitch#{uuid}.png")
+    send_file 'lib/assets/glitched/new_image.png', type: 'image/png', status: 202
```
Comment (dtsehl) [inline app/controllers/images_controller.rb:19]: The file path could be set up as an environment variable but this works too!

File: app/views/search/show.html.erb [hunk 2]
```diff
@@ -1,13 +1,18 @@
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- Add icon library -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (dtsehl) [inline app/views/search/show.html.erb:4]: I just looked up Font Awesome - it helps a ton with styling!

**Review by dtsehl — APPROVED** [review]

File: app/controllers/images_controller.rb [hunk 3]
```diff
@@ -13,6 +15,13 @@ def create
     redirect_to search_show_path(api_image)
   end
 
+  def download
+    # uuid = SecureRandom.uuid
```
Comment (janegreene) [inline app/controllers/images_controller.rb]: Can we remove the comments?

File: config/routes.rb [hunk 4]
```diff
@@ -13,6 +13,7 @@
   get '/about', to: 'developers#index'
   get '/500', to: 'application#server_errors'
   get '/400', to: 'application#search_errors'
+  match 'download', to: 'images#download', as: 'download', via: :get
```
Comment (janegreene) [inline config/routes.rb:16]: Oh neat! How does this specialness work?

**Review by janegreene — COMMENTED** [review]
Awesome, very exciting style improvements! We can remove the comments before we push if they are not used anymore.

File: app/controllers/images_controller.rb (hunk 3 above)
Comment (dtsehl) [reply app/controllers/images_controller.rb]: I just removed the comments and committed.

**Review by dtsehl — APPROVED** [review]

**Pull request merged**